sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/CreateNewEventPage.java

```diff
@@ -57,4 +58,36 @@
   public static final By CITY_INPUT = By.cssSelector(".popupContent #city");
   public static final By GPS_LATITUDE_INPUT = By.cssSelector(".popupContent #latitude");
   public static final By GPS_LONGITUDE_INPUT = By.cssSelector(".popupContent #longitude");
+  public static final By PRIMARY_MODE_OF_TRANSMISSION =
+      By.cssSelector(".popupContent #diseaseTransmissionMode");
+  public static final By NOSOCOMIAL = By.cssSelector(".popupContent #nosocomial");
+  public static final By EVENT_INVESTIGATION_START_DATE =
+      By.cssSelector(".popupContent #eventInvestigationStartDate");
+  public static final By EVENT_INVESTIGATION_END_DATE =
+      By.cssSelector(".popupContent #eventInvestigationEndDate");
+  public static final By SOURCE_MEDIA_WEBSITE = By.cssSelector(".popupContent #srcMediaWebsite");
+  public static final By SOURCE_MEDIA_NAME = By.cssSelector(".popupContent #srcMediaName");
+  public static final By SOURCE_MEDIA_DETAILS = By.cssSelector(".popupContent #srcMediaDetails");
+  public static final By SOURCE_FIRST_NAME = By.cssSelector(".popupContent #srcFirstName");
+  public static final By SOURCE_LAST_NAME = By.cssSelector(".popupContent #srcLastName");
+  public static final By SOURCE_TEL_NO = By.cssSelector(".popupContent #srcTelNo");
+  public static final By SOURCE_EMAIL = By.cssSelector(".popupContent #srcEmail");
+  public static final By SOURCE_INSTITUTIONAL_PARTNER =
+      By.cssSelector(".popupContent #srcInstitutionalPartnerType div");
+  public static final By SOURCE_INSTITUTIONAL_PARTNER_DETAILS =
+      By.cssSelector(".popupContent #srcInstitutionalPartnerTypeDetails");
+  public static final By SPECIFY_OTHER_EVENT_PLACE =
+      By.cssSelector(".popupContent #typeOfPlaceText");
+  public static final By MEANS_OF_TRANSPORT = By.cssSelector(".popupContent #meansOfTransport div");
+  public static final By CONNECTION_NUMBER = By.cssSelector(".popupContent #connectionNumber");
+  public static final By TRAVEL_DATE = By.cssSelector(".popupContent #travelDate");
+  public static final By FACILITY_CATEGORY = By.cssSelector(".popupContent #typeGroup div");
+  public static final By FACILITY_TYPE = By.cssSelector(".popupContent #facilityType div");
+  public static final By FACILITY = By.cssSelector(".popupContent #facility div");
+  public static final By MEANS_OF_TRANSPORT_DETAILS =
+      By.cssSelector(".popupContent #meansOfTransportDetails");
+  public static final By DISEASE_NAME = By.cssSelector(".popupContent #diseaseDetails");
+  public static final By REPORT_DATE = By.cssSelector(".popupContent #reportDateTime input");
+  public static final By NEW_EVENT_CREATED_MESSAGE =
+      By.xpath("//*[contains(text(),'New event created')]");
```

Comment:
Please add to the final of the variable name: `INPUT,  COMBOBOX or OPTIONS` depends on the selector.